arroyo-console/src/components/QueryCatalogue.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useContext } from 'react';
+import {
+  Drawer,
+  DrawerBody,
+  DrawerCloseButton,
+  DrawerContent,
+  DrawerHeader,
+  Flex,
+  Popover,
+  PopoverArrow,
+  PopoverBody,
+  PopoverCloseButton,
+  PopoverContent,
+  PopoverHeader,
+  PopoverTrigger,
+  Text,
+} from '@chakra-ui/react';
+import ExampleQueryCard from './ExampleQueryCard';
+import { exampleQueries } from '../lib/example_queries';
+import { TourContext, TourSteps } from '../tour';
+
+export interface QueryCatalogueProps {
+  isOpen: boolean;
+  onClose: () => void;
+  setQuery: (s: string) => void;
+}
+
+const QueryCatalogue: React.FC<QueryCatalogueProps> = ({ isOpen, onClose, setQuery }) => {
+  const { tourStep, disableTour } = useContext(TourContext);
+
+  return (
+    <Drawer isOpen={isOpen} placement="right" onClose={onClose} size={'xl'}>
+      <DrawerContent padding={4} bg={'gray.600'}>
+        <DrawerCloseButton onClick={onClose} />
+        <DrawerHeader bg={'gray.600'}>
+          Query Catalogue
```

Comment:
I think we can find a better name than catalogue for this. I expect that a catalogue is a place you create saved queries. I think it's a bit confusing that we're using it to mean a set of example queries.

Maybe just "Example queries"?

Reply:
Sure, done.